app-devel/patatt/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=patatt
+PKGSEC=devel
+PKGDEP="git setuptools pynacl"
```

Comment:
Shouldn't `git` and `setuptools` be build-time dependencies?

Reply:
Moved setuptools to build-time dependency. Git is kept.